<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::StateMachineInspector</class>
 <widget class="QWidget" name="GammaRay::StateMachineInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSplitter" name="splitter_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="stateMachinesView"/>
          <widget class="QTreeView" name="singleStateMachineView"/>
          <widget class="QTreeView" name="transitionView"/>
         </widget>
        </item>
       </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
